plugins/extend-give-wp/bootstrap.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Extend GiveWP
+ *
+ * @package     spiralWebDB\ExtendGiveWP
+ * @author      Robert A. Gadon
+ * @license     GPL-2.0+
+ *
+ * @wordpress-plugin
+ * Plugin Name: Extend GiveWP
+ * Plugin URI:  https://github.com/rgadon107/cornerstone/plugins/extend-give-wp/
+ * Description: This plugin extends the Give donation plugin by registering custom functions to it's hooks.
+ * Version:     1.0.0
+ * Author:      Robert A. Gadon
+ * Author URI:  https://spiralwebdb.com
+ * Text Domain: extend-give
+ * License:     GPL-2.0+
+ * License URI: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+namespace spiralWebDB\ExtendGiveWP;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit( 'Cheatin&#8217; uh?' );
+}
+
+define( 'EXTEND_GIVE_URL', plugin_dir_url( __FILE__ ) );
```

Comment:
Is this used/needed? If yes, instead of defining a constant that clutters the global space, use a private function and memoization (via `static` variable): `_get_plugin_url()`.

Reply:
@hellofromtonya I had a hunch you would spot this! Just as I was finishing my last commit, I saw the constant too, and realized it wasn't used anywhere in the plugin. It's a legacy from the `Sandbox-Workspace` plugin. Since it's not in use and not currently needed, I will remove it.

Reply:
@hellofromtonya See commit [48b6c6f](https://github.com/rgadon107/cornerstone/pull/52/commits/48b6c6f2f890300648d8ef55045ddd1fb28826d2). The constant was removed.